bindata/v3.11.0/openshift-controller-manager/image-import-controller-clusterrole.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: system:openshift:openshift-controller-manager:image-import-controller
+rules:
+- apiGroups:
+  - ""
```

Comment:
afaik you don't need the apiGroup `""` for imagestreams

Reply:
That's probably a legacy of using non-groupified APIs from way back. @deads2k is it safe for us to remove?

Reply:
> That's probably a legacy of using non-groupified APIs from way back. @deads2k is it safe for us to remove?

I suggest keeping it for the purposes of this PR.